### ISSUE TITLE
Add option to disable all GUI transitions/animations

### ISSUE THREAD

**Issue by tomasz1986** [canton7/SyncTrayzor#521]
I wonder whether it would be possible to add an option to disable all transitions / animations in the GUI.

Personally, transitions and animations make me physically feel dizzy, so I always try to disable them whenever possible.

In addition, they also cause slowdowns when interacting with the GUI on a remote computer, where the connection is too slow to transmit such screen changes smoothly. The same is true for a virtual machine with no hardware acceleration available, i.e. the GUI is very choppy there.

Thank you very much in advance.

**Comment by canton7** [comment]
Those are part of Syncthing's web UI - nothing to do with SyncTrayzor I'm afraid. Please ask [on the forum](https://forum.syncthing.net)

**Comment by tomasz1986** [comment]
Thank you very much!

I created a gui folder in SyncTrayzor\data\syncthing following the official FAQ ([https://docs.syncthing.net/users/faq.html#i-don-t-like-the-gui-or-the-theme-can-it-be-changed](https://docs.syncthing.net/users/faq.html#i-don-t-like-the-gui-or-the-theme-can-it-be-changed)).

In the theme.css, I have added my usual CSS settings which I use to override animations and transitions in my browsers on all websites. I use these, as simply setting them to `none` break things quite often.

```
*,
:before,
:after {
  animation-delay: 0s !important;
  animation-duration: 0s !important;
  animation-iteration-count: 1 !important;
  transition-delay: 0s !important;
  transition-duration: 0s !important;
}
```

No more animations and transitions in Syncthing :relaxed:.
